### PR TITLE
[EDA] - Create NEW Education History Object

### LINKED ISSUE #1266: Create new Education History Object
**Note: FOR INTERNAL USE ONLY. For support questions, please use the EDA Chatter group in [The Power of Us Hub](https://powerofus.force.com).
For feature requests, please use the Ideas tab in [The Power of Us Hub](https://powerofus.force.com).**

* **Do you want to request a *feature* or report a *bug*?**
Feature

* **What is the current behavior?**
The Education History object is new and therefore, there is no current functionality.

* **What is the expected behavior?**
Addition of this object will allow an Institution to follow a learner throughout their student journey and will play a key role in providing a 360 degree view of the learner. 

* **What is the motivation / use case for changing the behavior?**
The University application process relies heavily on thoroughly understanding an applicant’s education history; tracking every detail from post-secondary to other colleges as well as the education history of the applicant’s parents. Understanding the path that led an applicant to show interest in an educational institution as well as which paths most commonly resulted in an offer letter (and acceptance) are key data points in understanding and analyzing an educational institution’s recruitment funnel. 

    Education History, however, is also important beyond the recruiting and admission process. Once the applicant accepts admission to an educational institution and becomes a student, their education history up to that point is valuable for future conversations with their adviser. Once the student graduates and becomes alumni, knowing the entire education history of each individual can be important for targeted donation drives or mentorship programs. 

* **Acceptance Criteria**

  - **Given**: Visibility and access to any/all Education history records
  - **When**: A Contact in my EDA org has an Education History 
  - **Then**:  I can see a list of the student’s Education History showing me all the schools they attended, the start and end dates. Accessible in related lists and customizable page layouts. 

  - **Given**: Visibility and Access to any/all Education History Records
  - **When**: I go to a Contact’s Education History Related List
  - **Then**:  I can see a a learner's entrance and exit dates to the Institution, and the total time (duration) the Contact spent at the Institution.

  - **Given**: Visibility and Access to any/all Education History Records
  - **When**: Accessing the Contact’s Education History Record(s)
  - **Then**:  I can see the Contact's Class Rank, GPA Scale Reporting, Decile/Quintile/Quartile ranking method, Rank Weighted, Cumulative GPA, GPA Scale, and GPA Weighted for each institution listed.

  - **Given**: An Education History record on a Contact, I can go to the Account Lookup for the educational institution
  - **When**: It is available off Education History
  - **Then**:  Peruse the Account/account details like any other educational institution account in my org.

### PR DESCRIPTION
**Pull request by m-kinkade**
# Critical Changes

# Changes
## New Education History Object
Understanding a student’s previous educational experience plays an important role in the recruiting and admissions process, but also has other useful applications. Check out our new Education History object to see how it can help you follow students throughout their educational journeys. For example:

- Track alumni historical data for targeted mentorship or donation programs.
- Track the time a student’s spent at other schools.
- Track a student’s educational history (including class ranking and grade point averages) to help guide their decisions.

To easily access this new object, manually add the Education History related list to your Contact page layout. Also consider customizing the related list, list views, and compact layouts. For information, check out [Display a New Custom Object After a Product Release](https://powerofus.force.com/s/article/EDA-Display-Cust-Obj-Post-Rel).

# Issues Closed
Closes #1266 

# New Metadata
**Custom Objects**
Education_History__c.object

**Layouts**
Education_History__c-EDA Education History Layout.layout

**Tabs**
Education_History__c.tab

**Object Translations**
Education_History__c-ca.objectTranslation
Education_History__c-en_GB.objectTranslation
Education_History__c-en_US.objectTranslation
Education_History__c-es.objectTranslation
Education_History__c-es_MX.objectTranslation
Education_History__c-fr.objectTranslation

**Trigger**
TDTM_EducationHistory.trigger
TDTM_EducationHistory.trigger-meta.xml

# Deleted Metadata

# Testing Notes
https://salesforce.quip.com/t4GbAhrpS9wW

### CODE REVIEW

**Comment by kevinswiggum** [comment]
@m-kinkade Should page layout(s) be included this PR? cc @MatthewBlanski

**Comment by m-kinkade** [comment]
> @m-kinkade Should page layout(s) be included this PR? cc @MatthewBlanski

@kevinswiggum yes, they will be.  I'll be adding them shortly.

File: src/objects/Education_History__c.object [hunk 1]
```diff
@@ -0,0 +1,632 @@
...
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Date</type>
+    </fields>
+    <fields>
+        <fullName>Time_At_Institution__c</fullName>
+        <description>The total amount of time a Contact spent at the Educational Institution, determined by the Start Date and End Date.</description>
+        <externalId>false</externalId>
+        <formula>IF(
+    NOT(ISBLANK(End_Date__c)) &amp;&amp; NOT(ISBLANK(Start_Date__c)) &amp;&amp; (End_Date__c > Start_Date__c),
```
Comment (MatthewBlanski) [inline src/objects/Education_History__c.object:570]: @m-kinkade While this solution is effective, it is not precise. It'll start to slip over large amounts of time. We may have to find some manner of compromise that excludes days as a significant figure.

**Comment by m-kinkade** [comment]
@MatthewBlanski changes discussed this morning have been incorporated.  PR is now ready for review :)

**Pull request merged**